tendermint-testgen/src/helpers.rs

```diff
@@ -0,0 +1,83 @@
+//! Helper functions
+
+use serde::de::DeserializeOwned;
+use simple_error::*;
+use std::io::{self, Read};
+use tendermint::{vote, signature::Signature, amino_types};
+use signatory_dalek::Ed25519Verifier;
+use signatory::signature::Verifier;
+
+#[macro_export]
```

Comment:
Some documentation for tihs would be helpful.

Reply:
done

Reply:
Thanks!